node/tests/Unison/Test/Index.hs

```diff
@@ -59,12 +60,13 @@ nextKeyAfterRemoval :: RandomGen r => Prereqs r -> Assertion
 nextKeyAfterRemoval (genVar, bs) = do
   ident <- makeRandomId genVar
   db <- KVS.load bs ident
-  KVS.insert (pack "1") (pack "k1", pack "v1") db
```

Comment:
FYI, I'd add `{-# Language OverloadedStrings #-}` and you can skip putting `pack` everywhere.